### PR TITLE
Use try with resources, improve logging

### PR DESCRIPTION
**Pull request by blizniukov**

### CODE REVIEW

File: src/main/java/io/github/redouane59/twitter/helpers/AbstractRequestHelper.java [hunk 1]
```diff
@@ -42,7 +43,7 @@ protected AbstractRequestHelper(TwitterCredentials twitterCredentials, OAuth10aS
   }
 
   public static void logApiError(String method, String url, String stringResponse, int code) {
-    LOGGER.error("(" + method + ") Error calling " + url + " " + stringResponse + " - " + code);
+    LOGGER.error("({}}) Error calling {} {} - {}", method, url, stringResponse, code);
```
Comment (redouane59) [inline src/main/java/io/github/redouane59/twitter/helpers/AbstractRequestHelper.java:46]: is it normal to have one opening and two closing ?

**Comment by redouane59** [comment]
Hey, globally, why should we use brackets instead of variable in message ? Is it a good practice ?

File: src/main/java/io/github/redouane59/twitter/helpers/AbstractRequestHelper.java (hunk 1 above)
Comment (blizniukov) [reply src/main/java/io/github/redouane59/twitter/helpers/AbstractRequestHelper.java]: Thanks, fixed

**Comment by blizniukov** [comment]
> Hey, globally, why should we use brackets instead of variable in message ? Is it a good practice ?

If short - yes. It is good practice.
https://sonarcloud.io/organizations/default/rules?open=java%3AS2629&q=S2629

Here is link to sonarcloud, where more info about this.
If you accept, then i will edit all log statements.

I also have some ideas:)

**Comment by redouane59** [comment]
Awesome, thanks for the source. Yes don't hesitate, this project is opensource so feel free to improve whatever you want :)

**Pull request merged**